src/main/sass/_page-navigation.scss

```diff
@@ -0,0 +1,105 @@
+.previous-next-navigation {
```

Comment:
Hi, had a look at this PR in a bit more detail after our discussion.

I think re-using the naming convention (`<div class="govuk-previous-and-next-navigation">`) would be a bad idea, as it might suggest a direct link between the code-bases, where one doesn't exists, introducing confusion. Changing the root scope/class name, and keeping the rest of the markup/CSS, would be :+1:, possible with a "based on <git link> comment".

I'd recommend using the component markup/CSS over other GOV.UK implementations as it's likely to be more modern/polished (for example, the component has better accessibility support). That would also be the best starting point for adding to elements.